dev/tests/integration/framework/Magento/TestFramework/Application.php

```diff
@@ -557,6 +558,17 @@ private function copyAppConfigFiles()
             }
         }
     }
+    
+    /**
+     * Copies global configuration file from the tests folder (see TESTS_GLOBAL_CONFIG_FILE)
+     *
+     * @return void
+     */
+    private function copyGlobalConfigFile()
+    {
+        $targetFile = $this->_configDir . '/config.local.php';
```

Comment:
I don't think config.local.php is the correct file.

https://github.com/magento/magento2/issues/27864

Reply:
![image](https://user-images.githubusercontent.com/1639941/126016759-9e341cdd-dfd0-44a1-af65-b66e8e28dc93.png)
As far as I understand, it's no longer supported since `2.2`